upup/pkg/fi/nodeup/command.go

```diff
@@ -313,6 +313,13 @@ func evaluateSpec(c *api.Cluster) error {
 		return err
 	}
 
+	if c.Spec.KubeProxy != nil {
```

Comment:
Is this even reachable?  Nit pick that does not need to be handled in the PR.  Up to you.  Will let you self merge